# Phylogenetic Biology - Final Project

## Guidelines - you can delete this section before submission

This is a stub for your final project. Edit/ delete the text in this readme as needed.

There are two ways you can use this document:  
- You can download this file to a folder on your computer, edit this document and add other files (data, code, etc), and then zip up and submit the folder on canvas.
- You can for the [repository](finalproject) containing this document on gitub. Then commit and push your canges to the repository, and submit a link to the repository on canvas.

Github is a great way to work on projects, but also has a steep initial learning curve.


Some guidelines and tips:

- Use the stubs below to write up your final project. Alternatively, if you would like the writeup to be an executable document (with [knitr](http://yihui.name/knitr/), [jupytr](http://jupyter.org/), or other tools), you can create it as a separate file and put a link to it here in the readme.

- For information on formatting text files with markdown, see https://guides.github.com/features/mastering-markdown/ . You can use markdown to include images in this document by linking to files in the repository, eg `![GitHub Logo](/images/logo.png)`.

- The project must be entirely reproducible. In addition to the results, the repository must include all the data (or links to data) and code needed to reproduce the results.

- If you are working with unpublished data that you would prefer not to publicly share at this time, please contact me to discuss options. In most cases, the data can be anonymized in a way that putting them in a public repo does not compromise your other goals.

- Paste references (including urls) into the reference section, and cite them with the general format (Smith at al. 2003).

OK, here we go.

# Title of my project

## Introduction and Goals

The goal of my project is to answer the question, What is...?
A great deal of my time is spent with non biting midges Chironomidae in my lab. Most of our samples have been clean freshwater samples from lakes. However I want to explore genetic differences in these species and see if this is leading to evolutionary differences. As there has been observed morphological differences. Hopefully my project shows that if there is clear clustering between the groups then there is limited genetic divergence between the populations and cause of gene flow/similar ecological pressures across environments. Or if there are two distance clusters between the groups that may imply the midge populations adapted differently. (If I have to pick a more specific group I would like to pick Tanypodinae).

The methods I will use to do this are...
Data Collection: I hope to use BOLD systems and or Genbank
Sequence Alignment: I plan to look at the COI gene but I have been reading papers on midge phylogenies to better understand what else I could look for. Like  nuclear 28S and CAD.
Model Selection: with this data I want to use R and the other programs on the cluster we have been shown in class. Like I think I want to construct the tree in IQ TREE then import it to R.
Tree Construction: Most of the Chironomidae phylogeny papers I have read have used Bayesian and maximum likelihood phylogenetic inferences.

The data I will use are (my own data/ data publicly available at YYY/ simulations)
I will try to use BOLD systems public data, and I would like to possibly consider using some of my lab data but that will probably need to be morphological if I do. 
## Methods

The tools I used were... See analysis files at (links to analysis files).

## Results

The tree in Figure 1...

## Discussion

These results indicate...

The biggest difficulty in implementing these analyses was...

If I did these analyses again, I would...

## References

